operator/src/main/resources/Operator.properties

```diff
@@ -54,9 +54,10 @@ WLSKO-0052=Existing managed server Service is correct for WebLogic domain with U
 WLSKO-0053=Creating cluster Service for WebLogic domain with UID: {0}. Cluster name: {1}.
 WLSKO-0054=Replacing cluster Service for WebLogic domain with UID: {0}. Cluster name: {1}.
 WLSKO-0055=Existing cluster Service is correct for WebLogic domain with UID: {0}. Cluster name: {1}.
-WLSKO-0056=Creating domain config map, {0}, for namespace: {1}.
-WLSKO-0057=Replacing domain config map, {0}, for namespace: {1}.
-WLSKO-0058=Existing domain config map, {0}, is correct for namespace: {1}.
+WLSKO-0056=Creating config map, {0}, for namespace: {1}.
+WLSKO-0057=Replacing config map, {0}, for namespace: {1}.
+WLSKO-0171=Patching config map, {0}, for namespace: {1}.
```

Comment:
Not a big deal and we're grouping config map messages together but should we move this down to the end for the sake of ordering?

Reply:
I wasn't really sure how to handle cases like this. It would have been better, perhaps to have left gaps in the numbering.

OTOH, maybe we don't even need to preserve the numbers from one commit to the next. I just don't know the intent.